### DISCUSSION TITLE
error with patch call, or is it me?

### DISCUSSION THREAD

**Discussion by cfritze64** [johnlindquist/kit#299] [Errors] [answered]
I'm making a patch call to the Zoom API and getting this error, which I think comes directly from the call:

[2021-05-30 11:39:42.592] [warn]  UNCAUGHT EXCEPTION: TypeError: Converting circular structure to JSON
    --> starting at object with constructor 'ClientRequest'
    |     property 'socket' -> object with constructor 'TLSSocket'
    --- property '_httpMessage' closes the circle

my call is:

  const response = await patch(`${ZOOM_API_ENDPOINT}/meetings/${theID}`, JSON.stringify(aMeeting), options);

I'm reasonably sure this is formatted correctly, as I'm using similar APIs in that system with success, AND my zoom meeting is in fact correctly updated. 

the next instruction in my code is 
console.log(response) 

so perhaps the problem is with what patch returns?

the relevant page of the Zoom API def is https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingupdate

**Comment by cfritze64** [comment, marked as answer]
uh, nevermind. Apparently I biffed the debugging. Edited my function some more and all is well...

**Reply by Ridermansb** [reply]
I'm having the same problem

> UNCAUGHT EXCEPTION: TypeError: Converting circular structure to JSON
